app/src/main/java/com/example/training/HomeActivity.java

```diff
@@ -0,0 +1,52 @@
+package com.example.training;
+
+import androidx.appcompat.app.AppCompatActivity;
+
+import android.content.Context;
+import android.content.Intent;
+import android.os.Bundle;
+import android.widget.TextView;
+
+public class HomeActivity extends AppCompatActivity {
+
+    public static String USER_FULL_NAME = "userName";
```

Comment:
create seperate AppConstant.java interface and place this line in this file

Reply:
created separate interface for constants